packages/manager/modules/telecom-dashboard/src/identity-check/telecom-dashboard-identity-check.service.js

```diff
@@ -0,0 +1,40 @@
+const inProgressStatuses = ['doing', 'todo', 'waiting_for_customer'];
+
+export default class {
+  /* @ngInject */
+  constructor($http) {
+    this.$http = $http;
+  }
+
+  isProcedureRequired() {
+    return this.$http
+      .get('/telephony/procedure/required')
+      .then(({ data: required }) => required);
+  }
```

Comment:
i am not able to find the usage of this function, are you sur i used ?
Also, if it's require, it's better to do this into the routing file.

Reply:
@mohammed-zahaf Use in packages/manager/modules/telecom-dashboard/src/identity-check/message/telecom-dashboard-identity-check-message.controller.js

Reply:
Thanks for replay, i don't care it's into a service.
My bad :( .

thanks